ui/raidboss/data/04-sb/eureka/eureka_hydatos.txt

```diff
@@ -16,7 +16,7 @@ hideall "--sync--"
 
 # Note: This checks that Art's auto hits a target close enough for you to see.
 # You can see Owain's auto abilities, but the target is listed as empty.
-1002.5 "--sync--" sync / 1[56]:[^:]:Art:3956:/ window 1500,0
+1002.5 "--sync--" sync / 1[56]:[^:]*:Art:3956:[^:]*:[^:]*:/ window 1500,0
```

Comment:
Mmm sorry, but I think that's not it either.  (And you also need to handle the Owain sync below as well.)

The format is `[timestamp] ActionEffect 15:[sourceId]:[source]:[id]:[ability]:[targetId]:[target]:`

So, I think you'd need: `1002.5 "--sync--" sync / 1[56]:[^:]*:Art:3956:[^:]*:[^:]+:[^:]+:/ window 1500,0`

Reply:
That's fine, I'm just starting some scheduled  in-game stuff so I'll be back at this a little later tonight.